### PR TITLE
refactor(shared-data): update H-S schema v6 commands

### LINKED ISSUE #10138: PE, App, PD, Step-Generation, Shared-Data: Rename H-S module schema v6 commands #10137
# Overview

Rename all H-S command names and parameters outlined in [the schema change doc](https://opentrons.atlassian.net/wiki/spaces/RPDO/pages/3184656624/Modules+JSON+SchemaV6+commands+vs.+PAPIv2+v3+method+names) across the entire stack

# Implementation details

Make changes across PE, PD, SG, all types, and the actual schema v6 spec.

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview

This PR updates JSON schema v6 H-S commands to match our updated interface. All frontend + backend changes to accommodate the schema change should be made in this PR.

Will close https://github.com/Opentrons/opentrons/issues/10138

Co-authored by @jbleon95 and @sanni-t 

# Changelog

-  Update H-S schema v6 commands

# Review requests

# Risk assessment
Med

### CODE REVIEW

File: shared-data/protocol/schemas/6.json [hunk 1]
```diff
@@ -900,7 +900,7 @@
             "properties": {
               "key": { "type": "string" },
               "commandType": {
-                "enum": ["heaterShakerModule/stopShake"]
+                "enum": ["heaterShaker/stopShake"]
```
Comment (sanni-t) [inline shared-data/protocol/schemas/6.json]:
```suggestion
                "enum": ["heaterShaker/deactivateShaker"]
```

File: shared-data/protocol/schemas/6.json [hunk 2]
```diff
@@ -915,7 +915,7 @@
             "properties": {
               "key": { "type": "string" },
               "commandType": {
-                "enum": ["heaterShakerModule/openLatch"]
+                "enum": ["heaterShaker/openLatch"]
```
Comment (sanni-t) [inline shared-data/protocol/schemas/6.json]:
```suggestion
                "enum": ["heaterShaker/openLabwareLatch"]
```

File: shared-data/protocol/schemas/6.json [hunk 3]
```diff
@@ -930,7 +930,7 @@
             "properties": {
               "key": { "type": "string" },
               "commandType": {
-                "enum": ["heaterShakerModule/closeLatch"]
+                "enum": ["heaterShaker/closeLatch"]
```
Comment (sanni-t) [inline shared-data/protocol/schemas/6.json]:
```suggestion
                "enum": ["heaterShaker/closeLabwareLatch"]
```

**Review by smb2268 — APPROVED** [review]

File: api/src/opentrons/protocol_engine/commands/heater_shaker/deactivate_shaker.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+            module_id=params.moduleId
+        )
+
+        # Allow propagation of ModuleNotAttachedError.
+        hs_hardware_module = self._equipment.get_module_hardware_api(
+            hs_module_substate.module_id
+        )
+
+        if hs_hardware_module is not None:
+            await hs_hardware_module.set_speed(rpm=0)
```
Comment (sanni-t) [inline api/src/opentrons/protocol_engine/commands/heater_shaker/deactivate_shaker.py]: The h/s hardware control class now has a `deactivate_heater` method. Let's use that.
```suggestion
            await hs_hardware_module.deactivate_shaker()
```

File: api/src/opentrons/protocol_engine/commands/heater_shaker/set_and_wait_for_shake_speed.py [hunk 5]
```diff
@@ -11,26 +11,26 @@
...
     moduleId: str = Field(..., description="Unique ID of the Heater-Shaker Module.")
     # TODO(mc, 2022-02-24): for set temperature we use `temperature` (not `celsius`)
     # but for shake we use `rpm` (not `speed`). This is inconsistent
     rpm: float = Field(..., description="Target speed in rotations per minute.")
 
 
-class SetTargetShakeSpeedResult(BaseModel):
-    """Result data from setting a Heater-Shaker's shake speed."""
+class SetAndWaitForShakeSpeedResult(BaseModel):
+    """Result data from setting and wait for a Heater-Shaker's shake speed."""
```
Comment (sanni-t) [inline api/src/opentrons/protocol_engine/commands/heater_shaker/set_and_wait_for_shake_speed.py]:
```suggestion
    """Result data from setting and waiting for a Heater-Shaker's shake speed."""
```

**Review by sanni-t — APPROVED** [review]
Sorry for the late review! I totally missed the notifications on this PR. 
Python changes look good. Just one non-naming related change I noticed (since we're touching that code anyway..)

File: api/src/opentrons/protocol_engine/commands/heater_shaker/wait_for_temperature.py [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+
+WaitForTemperatureCommandType = Literal["heaterShaker/waitForTemperature"]
+
+
+class WaitForTemperatureParams(BaseModel):
+    """Input parameters to wait for a Heater-Shaker's target temperature."""
+
+    moduleId: str = Field(..., description="Unique ID of the Heater-Shaker Module.")
+    celsius: Optional[float] = Field(None, description="Target temperature in °C.")
```
Comment (sanni-t) [inline api/src/opentrons/protocol_engine/commands/heater_shaker/wait_for_temperature.py]:
```suggestion
    celsius: Optional[float] = Field(None, description="Target temperature in °C. If not specified, will default to the module's target temperature. Specifying a celsius parameter other than the target temperature could lead to unpredictable behavior and hence is not recommended for use. This parameter can be removed in a future version without prior notice.")
```

**Pull request merged**